pkgs/top-level/python-packages.nix

```diff
@@ -32014,6 +32014,40 @@ EOF
   yenc = callPackage ../development/python-modules/yenc {
   };
 
+  zeep = buildPythonPackage rec {
```

Comment:
We have a new policy for python to put python packages into a separate file and only reference them here:

See also:

```
$ chromium "file://$(nix-build --no-out-link doc)//share/doc/nixpkgs/manual.html#contributing-guidelines"
```